utils/state-machine.js

```diff
@@ -5,25 +5,68 @@ const setReservationType = assign({
     return event.value;
   },
 });
+
+const removeReservationType = assign({
+  reservationType: () => null,
+});
 const setCertificationType = assign({
   certificationType: (context, event) => event.value,
 });
+
+const removeCertificationType = assign({
+  certificationType: () => null,
+});
+
+const setDeepestDive = assign({
+  deepestDive: (context, event) => event.value,
+});
+
+const removeDeepestDive = assign({
+  deepestDive: () => null,
+});
+
+const setLastDive = assign({
+  lastDive: (context, event) => event.value,
+});
+
+const removeLastDive = assign({
+  lastDive: () => null,
+});
```

Comment:
given that these are pretty simple, lets just go ahead and set them in the actual state in the future (just so you know)